pos_order_to_sale_order/README.rst

```diff
@@ -0,0 +1,130 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=======================
+POS Order To Sale Order
+=======================
+
+
+This module extends the functionality of point of sale to allow you create
+Sale Orders from the Point of sale.
+
+It adds a new button, below the current bill on the point of sale front end UI,
+that allows you to create a Sale order, based on the current PoS Order lines
+and then to discard the current PoS Order.
+
+This module is usefull if you have some customers that come every day in your
```

Comment:
I think it is a bit limited: this is not the only case I know. Another potential usage is the possibility to have a tablet in a fair and take orders with a very simple interface
Maybe add "For example"